### ISSUE TITLE
Revisit ES Modules support

### ISSUE THREAD

**Issue by wildwestrom** [rasendubi/uniorg#24]
Looks like Next.js now has ES modules support; https://nextjs.org/blog/next-12
Gatsby still doesn't unfortunately... https://github.com/gatsbyjs/gatsby/discussions/31599

However, would you consider making it so that we can use ES modules now?
I don't think there's any workaround with Svelte, which uses ES modules exclusively.

I'm getting a little better at Javascript, so maybe I could help implement one of the solutions proposed in this article.
https://redfin.engineering/node-modules-at-war-why-commonjs-and-es-modules-cant-get-along-9617135eeca1

**Comment by rasendubi** [comment]
I would love to have dual cjs/mjs support, but it is more complicated than
described in the post.

1. unified ecosystem has moved to esm-only, not dual. And you cannot
provide a (dual) cjs module that works with esm-only.

2. While transitioning to esm-only, unified has also introduced other
breaking changes—the API has changed.

Supporting both cjs/mjs at this point would require maintaining two major
versions of uniorg in parallel, which I don't think is worth the effort.

**Comment by wildwestrom** [comment]
This may or may not be related.
In my svelte-kit project I get a warning.
```
$ npx svelte-kit dev

  SvelteKit v1.0.0-next.201

  local:   http://localhost:3000
  network: not exposed

  Use --host to expose server to other devices on this network

uniorg is incorrectly packaged. Please contact the package author to fix.
```

**Comment by bluwy** [comment]
@wildwestrom The message you get from Vite is a bug there, it should be fixed by https://github.com/vitejs/vite/pull/6053 soon. So I don't think it's related to this issue, though ES Modules support is still something nice to have.

**Comment by rasendubi** [comment]
uniorg v0.5.0 is now ESM-only 🎉
